pkg/csi/service/mounter/mounter_windows.go

```diff
@@ -218,28 +218,20 @@ func (mounter *csiProxyMounter) GetDiskNumber(ctx context.Context, diskID string
 	return "", errors.New("no matching disks found")
 }
 
-// IsLikelyMountPoint - If the directory does not exists, the function will return os.ErrNotExist error.
-// If the path exists, call to CSI proxy will check if its a link, if its a link then existence of target
-// path is checked.
+// IsLikelyMountPoint - If the directory does not exists, the function will return error.
+// If the path exists, will check if it is a symlink.
+// Note: Not using CSI proxy IsSymlink() function to check for symlink for Windows, as it tries to read the link.
 func (mounter *csiProxyMounter) IsLikelyNotMountPoint(path string) (bool, error) {
-	isExists, err := mounter.ExistsPath(mounter.Ctx, path)
+	stat, err := os.Lstat(path)
 	if err != nil {
-		return false, err
-	}
-
-	if !isExists {
-		return true, os.ErrNotExist
+		return true, err
 	}
 
-	response, err := mounter.FsClient.IsSymlink(mounter.Ctx,
-		&fs.IsSymlinkRequest{
-			Path: normalizeWindowsPath(path),
-		})
-	if err != nil {
+	if stat.Mode()&os.ModeSymlink != 0 {
 		return false, err
 	}
-	return !response.IsSymlink, nil
-	// TODO check if formatted else error out
+	return true, nil
+	//TODO check if formatted else error out
```

Comment:
Can you add some comments to the code?

We can make this local change first, but It will be great if you could also submit a PR to the csi-proxy repo.

Reply:
I have added note above as
// Note: Not using CSI proxy IsSymlink() function to check for symlink for Windows, as it tries to read the link.
Is that enough or shall I add some more comments ??

Reply:
What I meant is to add some comments before calling os.Lstat() and stat.Mode()&os.ModeSymlink != 0.

Reply:
done